files/es/web/javascript/reference/global_objects/object/object/index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Object() constructor
+slug: Web/JavaScript/Reference/Global_Objects/Object/Object
+page-type: javascript-constructor
+tags:
+  - Constructor
+  - JavaScript
+  - Object
+  - Reference
+browser-compat: javascript.builtins.Object.Object
```

Comment:
- Translate title
- Delete tags not necessary, they're taken from the `en-US` file
- Add `sourceCommit` that indicates the version of the `en-US` source

```suggestion
title: Constructor Object()
slug: Web/JavaScript/Reference/Global_Objects/Object/Object
l10n:
  sourceCommit: effd5de5e42bfe045c3bf44b2d7b14f4d6146785
```